src/index.html

```diff
@@ -20,8 +21,440 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <!-- #region HEADER -->
+  <body class="body">
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+          class="logo__img"
+        />
+      </a>
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```

Reply:
+